src/pages/dashboard/index.tsx

```diff
@@ -9,6 +9,7 @@ import { applications as getApplications } from "../../store/application-store/a
 import { getAllUsers } from "../../store/user-store/userActions";
 import { getEvents } from "../../store/event-store/eventActions";
 import { getInquiries } from "../../store/inquiry-store/inquiryAction";
+import MeetingScheduler from "./meeting_scheduler";
```

Comment:
Move this import after line no 8

Reply:
![image](https://user-images.githubusercontent.com/85297495/164370327-216395f4-df6e-4bcc-b025-f0f44fa59c6f.png)
Done..